web/themes/custom/novel/templates/fields/address-plain.html.twig

```diff
@@ -0,0 +1,19 @@
+{# Making sure plain address does not have other tags, other than div and br. #}
+
+{% set fields = [
+  given_name ~ " " ~ family_name,
+  organization,
+  address_line1,
+  address_line2,
+  dependent_locality.code,
+  locality.code ~ " " ~ postal_code ~ " " ~ administrative_area.code,
+  country.name
+] %}
+
+<div class="address" translate="no">
```

Comment:
Consider if this markup is relevant.

As far as I can tell the class it not specified in the design system and the `translate="no"` attribute is not specified there either.

As far as I can tell it is the first time we add it so I wonder if you think it is something that is relevant in this context specifically or we should look into in general.

Reply:
This is from the original address-plain file. I dont know why they've added it, but i assume there's a point, so i've just done it too.